website/pages/docs/single-page-applications.md

```diff
@@ -1,6 +1,44 @@
-# Single Page Applications
+# Single Page Applications (https://github.com/jaredpalmer/razzle/blob/master/examples/basic-spa/)
 
-In addition to universal/isomorphic appplications, Razzle can build single page (or client-only) applications. To do this, you can remove `index.js` and `server.js` then `index.html` file inside public folder at the end pass `--type=spa` to your `package.json`'s scripts like so:
+In addition to universal/isomorphic applications, Razzle can build single page (or client-only) applications. To do this, remove `index.js` and `server.js` from the src folder, and create an `index.html` file inside public folder with the following template:
```

Comment:
Inside the public

Reply:
@fivethreeo typo fixed. Lots of conflicts when rebasing so I branched off of next-awesome and made the change and force pushed. Also changed the PR base to next-awesome instead of master. Hope that works. 

Is it always this much work to update the documentation? Feels like for documentation contributors shouldn't have to do all that.

Thanks!